contents/blog/postmortem-rebrand.md

```diff
@@ -0,0 +1,92 @@
+---
+date: 2021-09-28
+title: Postmortem - How we rebranded PostHog in four weeks
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: cory-watilo
+featuredImage: ../images/blog/blog-generic-1.png
+featuredImageType: full
+categories: ["General", "Company & culture"]
+---
+We recently completed an entire rebrand of our website, from inception to completion, in less than a month. This sounds like an incredible feat of discipline - and in a way, it was. But the path to success wasn’t linear - it took us more than six months to get to this point, with lots of diversions along the way.
+
+Since we’re an open source company and our [communication is public](https://posthog.com/handbook/company/communication#public-by-default), it seemed like a public post-mortem was in order, despite how vulnerable this makes me feel as a designer.
+
+## Joining PostHog during a pivot
+When I joined PostHog in early 2021, we were focused on companies who needed to self-host their analytics. Soon afterwards we shifted focus to PostHog Cloud. Lately we’ve shifted back to self-hosted deployments and since the start of the year we’ve also grown into a platform that extends beyond just analytics.
+
+As a designer, my task was the effectively communicate all of this. However I also felt it wasn’t in my best interests to propose massive changes on day one. PostHog already had a [Graphic Designer](https://posthog.com/handbook/company/team#lottie-coxon-graphic-designer) and it was important to take the time to understand who I would be working with, especially because PostHog already had a defined visual style at that point. 
```

Comment:
```suggestion
As a designer, my task was to effectively communicate all of this. However, I also felt it wasn’t in my best interests to propose massive changes on day one. PostHog already had a [Graphic Designer](https://posthog.com/handbook/company/team#lottie-coxon-graphic-designer) and it was important to take the time to understand who I would be working with, especially because PostHog already had a defined visual style at that point. 
```